webextension/background/main.js

```diff
@@ -62,5 +63,19 @@ window.main = (function () {
     chrome.tabs.create({url: backend + "/shots"});
   });
 
+  communication.register("openShot", ({url, copied}) => {
```

Comment:
I suspect you've found some completely dead code, there was a point when the content worker was uploading the shot and then the background page opened the tab.